dive_sailthru_client/tests/test_integration.py

```diff
@@ -0,0 +1,83 @@
+from unittest import TestCase
+from nose.plugins.attrib import attr
+from dive_sailthru_client.client import DiveSailthruClient
+import os
+import datetime
+import tempfile
+import StringIO
+
+
+@attr('external')
+class TestDiveSailthruClientExternalIntegration(TestCase):
+    def setUp(self):
+        sailthru_key = os.getenv("SAILTHRU_API_KEY")
+        sailthru_secret = os.getenv("SAILTHRU_API_SECRET")
+        assert sailthru_key, "SAILTHRU_API_KEY env var must be defined"
+        assert sailthru_secret, "SAILTHRU_API_SECRET env var must be defined"
+        assert sailthru_key.lower().startswith("bc8"), "This test requires the DEV account Sailthru API key"
+        self.sailthru_client = DiveSailthruClient(sailthru_key, sailthru_secret)
+        self.test_email = 'eli+sailthru-client-integration-test@industrydive.com'
+        self.test_var_key = "sailthruclienttestvalue"
+
+    # def test_export_list(self):
```

Comment:
does this need to be removed from the file?

Reply:
Sure you can remove it. I thought about adding a similar test for export_list but decided it was too much for this PR